canonical-web-and-design/maas-ui
# Fix the message when the websocket closes

## Done
- Correctly get the url when the websocket disconnects.

## QA
- Visit a legacy url.
- Stop your maas websocket.
- In your console you should see a warning like: `WebSocket connection closed: ws://dev.local:8400/MAAS/ws`

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1252.
